src/RBDyn/NumericalIntegration.h

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2012-2019 CNRS-UM LIRMM, CNRS-AIST JRL
+ */
+
+#pragma once
+
+// includes
+// std
+#include <vector>
+
+// RBDyn
+#include <rbdyn/config.hh>
+#include <rbdyn/deprecated.hh>
+
+#include "Joint.h"
+
+namespace rbd
+{
+class MultiBody;
+struct MultiBodyConfig;
+
+/**
+ * Integrate a constant rotation acceleration.
+ * @param qi Initial orientation.
+ * @param wi Initial rotation speed.
+ * @param wD Constant acceleration.
+ * @param step Integration step.
+ * @param relEps Stopping criterion for the underlying Magnus expansion, relatively to the norm of its first term.
+ * @param absEps Absolute precision required on the underlying Magnus expansion.
+ * @param breakOnWarning If true, computation is stopped right after the underlying Magnus expansion when it returns
+ * a warning, and qi is returned, otherwise perform the full computation.
+ */
+RBDYN_DLLAPI std::pair<Eigen::Quaterniond, bool> SO3Integration(
+                                                      const Eigen::Quaterniond & qi,
+                                                      const Eigen::Vector3d & wi,
+                                                      const Eigen::Vector3d & wD,
+                                                      double step,
+                                                      double relEps = 1e-12,
+                                                      double absEps = std::numeric_limits<double>::epsilon(),
+                                                      bool breakOnWarning = false);
```

Comment:
`breakOnWarning` should be renamed since no warning is emitted (not to the user/developer).